### PR TITLE
[ Feature ]:: runjs included all actions 

### LINKED ISSUE #3420: Trigger actions from RunJS
### Is your proposal related to a problem?

Currently only 3 actions are availbale.

### Describe the solution you'd like
Add remaining actions
logout
show-alert
open-webpage
go-to-app
show-modal
close-modal
copy-to-clipboard
set-localstorage-value
generate-file

### If the feature is approved, would you be willing to submit a PR?
Yes

### PR DESCRIPTION
**Pull request by stepinfwd**
**actions include**
--------------------
- logout
- showModal
- closeModal
- setLocalStorage
- copyToClipboard
- generateFile
- goToApp
- showAlert

closes #3420

### CODE REVIEW

File: frontend/src/Editor/QueryManager/QueryManager.jsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ let QueryManager = class QueryManager extends React.Component {
         appId: props.appId,
         dataSources: props.dataSources,
         dataQueries: dataQueries,
+        appDefinition: props.appDefinition,
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/QueryManager/QueryManager.jsx:63]: I don't see the `appDefinition` used anywhere in the file. Can you please remove it if it is not needed?

File: frontend/src/Editor/Editor.jsx [hunk 2]
```diff
@@ -1493,6 +1493,8 @@ class Editor extends React.Component {
                             runQuery={this.runQuery}
                             dataSourceModalHandler={this.dataSourceModalHandler}
                             setStateOfUnsavedQueries={this.setStateOfUnsavedQueries}
+                            appDefinition={appDefinition}
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Editor.jsx:1496]: Can you check and remove 'appDefinition' prop if it is not used in `QueryManager.jsx` file?

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@stepinfwd Suggested a very minor change

File: frontend/src/Editor/Editor.jsx (hunk 2 above)
Comment (stepinfwd) [reply frontend/src/Editor/Editor.jsx:1496]: Hey appdefinition ( this for querymanager) is used in modalactions in `apputils.js` @kavinvenkatachalam

Comment (stepinfwd) [reply frontend/src/Editor/Editor.jsx:1496]: Hey appdefinition ( this for querymanager) is used in modalactions in `apputils.js` @kavinvenkatachalam

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Review by sherfin94 — APPROVED** [review]

**Pull request merged**